elements/pfe-tabs/src/pfe-tab.scss

```diff
@@ -3,90 +3,94 @@
 $LOCAL: tabs;
 
 $LOCAL-VARIABLES: (
-  BackgroundColor: transparent,
-  Color:           pfe-var(ui-disabled--text),
-  link:            pfe-var(link),
-  focus:           pfe-var(link--focus),
-  highlight:       pfe-var(ui-accent),
-  tab: (
-    FontSize:       pfe-var(font-size),
-    PaddingTop:     calc(#{pfe-var(container-padding)} * .666),
-    PaddingRight:   calc(#{pfe-var(container-padding)} * 1.5),
-    PaddingBottom:  calc(#{pfe-var(container-padding)} * .666),
-    PaddingLeft:    calc(#{pfe-var(container-padding)} * 1.5),
+  BackgroundColor:           transparent,
+  BackgroundColor--inactive: pfe-var(surface--lighter),
+  Color:                     pfe-var(text--muted),
+  BorderColor--hover:        $pf-color-black-400, 
+  BorderWidth:               pfe-var(ui--border-width--active),
+  tab: ( // to avoid collisions with panel
+    FontSize:                pfe-var(font-size),
+    Margin:                  0 0 calc( #{pfe-var(ui--border-width)} * -1),
+    PaddingTop:              pfe-var(container-padding),
+    PaddingBottom:           pfe-var(container-padding),
+    PaddingRight:            calc(#{pfe-var(container-padding)} * 2),
+    PaddingLeft:             calc(#{pfe-var(container-padding)} * 2),
+    TextTransform:           none,
   )
 );
 
+// TODO: These vars (focus & highlight) will be deprecated at 1.0  
+$backwards-compatibility: (
+  Color--focus:  pfe-local(focus, pfe-var(text)), 
+  BorderColor:   pfe-local(highlight, pfe-var(ui-accent)),
+);
+$LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, $backwards-compatibility);
+
+@include pfe-local-debug;
```

Comment:
// TODO: remove this when done @starryeyez024

Reply:
removed